pom.xml

```diff
@@ -306,6 +313,68 @@
               </environmentVariables>
             </configuration>
           </execution>
+          <execution>
+            <id>domain-socket-tests</id>
+            <goals>
+              <goal>integration-test</goal>
+            </goals>
+            <phase>integration-test</phase>
+            <configuration>
+              <includes>
+                <include>io/reactiverse/pgclient/it/UnixDomainSocketTest.java</include>
+              </includes>
+              <environmentVariables>
+                <PGHOST>/var/run/postgresql</PGHOST>
+                <PGDATABASE>postgres</PGDATABASE>
+                <PGUSER>postgres</PGUSER>
+                <PGPASSWORD>postgres</PGPASSWORD>
+              </environmentVariables>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>io.fabric8</groupId>
```

Comment:
can we try to use TestContainer instead ? https://www.testcontainers.org

Reply:
I have migrated it to `TestContainer`. By the way, this is a really nice tool!

Reply:
I haven't tried it yet but it looks promisinig for testing with docker